cmd/sealer/cmd/build.go

```diff
@@ -60,48 +69,114 @@ build with args:
 	sealer build -f Kubefile -t my-kubernetes:1.19.8 --build-arg MY_ARG=abc,PASSWORD=Sealer123 .
 `,
 	RunE: func(cmd *cobra.Command, args []string) error {
-		buildContext := args[0]
+		return buildSealerImage()
```

Comment:
Here you directly ignore the args, because you have not passed args into function buildSealerImage(). Then I am wondering if we could update `Args: cobra.MaximumNArgs(1),` to be `Args: cobra.NoArgs,`.